packages/error-reporting/test/unit/testLogger.js

```diff
@@ -52,9 +52,33 @@ describe('logger', function() {
       });
     });
     describe('Default log level', function() {
+      var oldLog;
+      var text;
+      beforeEach(function() {
+        oldLog = console.log;
+        text = '';
+        console.log = function() {
+          oldLog.apply(this, arguments);
+          for (var i=0; i<arguments.length; i++) {
+            text += arguments[i];
+          }
+        };
+      });
+     afterEach(function() {
```

Comment:
It looks like this is not indented the same as the other lines. I wonder if our linter is ignoring this file intentionally.